src/engines/evaluate-join-potential.ts

```diff
@@ -0,0 +1,71 @@
+import { FastNode } from './FastNode'
+import { propagateJoinMessages } from './join-propagation'
+import { evaluate, evaluateMarginalPure } from './evaluation'
+import { FastClique } from './FastClique'
+import { Formula } from './Formula'
+import { FastPotential } from './FastPotential'
+import { sum } from 'ramda'
+
+/** Construct the join of an arbitrary collection of variables in a Bayesian Network,
+ * conditioned on an optional set of parent variables.  Returns the potential function
+ * of the joint distribution over the head and parent variables subject to any evidence.
+ * @param nodes: The collection of nodes in the bayesian network
```

Comment:
Bayesian should be capitalized. See other locations for the same thing.

There's a bunch of back and forth in there between Bayesian Network and Bayes Network. Not sure if we need/want to be consistent.

Reply:
fixed